components/partials/fee-discounts/fees/tier.vue

```diff
@@ -86,22 +84,22 @@ export default Vue.extend({
       return stakedAmount.toFormat(0)
     },
 
-    feePaidAmount(): BigNumberInBase {
+    volume(): BigNumberInBase {
       const { tier } = this
 
-      if (!tier.feePaidAmount) {
+      if (!tier.volume) {
         return ZERO_IN_BASE
       }
 
       return new BigNumberInWei(
-        cosmosSdkDecToBigNumber(tier.feePaidAmount)
+        cosmosSdkDecToBigNumber(tier.volume)
       ).toBase(6 /* USDT */)
```

Comment:
Nitpick here, lets move 6 to the constant file for e.g. `USDT_DECIMAL_PLACE: number = 6` and use it here and remove the comment